docs/Datadog.Trace/README.md

```diff
@@ -0,0 +1,220 @@
+
+### Introduction of ISpan and IScope
+
+In version 2.0 the public `Tracer` API has been updated to expose `IScope` and `ISpan` instead of `Scope` and `Span`. In most cases, this should not require any changes to your code, but if you are currently using explicit types (instead of inferred types), then you will need to replace usages of `Scope` with `IScope` and `Span` with `ISpan`:
+
+```csharp
+using Datadog.Trace;
+
+// no changes required here
+using (var scope = Tracer.Instance.StartActive("my-operation")
+{
+}
+
+// No Longer compiles (incorrect usage of Scope)
+using (Scope scope = Tracer.Instance.StartActive("my-operation")
+{
+}
+
+// Correct usage with explicit type
+using (IScope scope = Tracer.Instance.StartActive("my-operation")
+{
+}
+```
+
+### Incorrect integration names are ignored
+
+In version 2.0, any changes made to an `IntegrationSettings` object for an unknown `IntegrationId` will not be persisted. Instead, a warning will be logged describing the invalid access.
+
+```csharp
+using Datadog.Trace;
+
+var tracerSettings = TracerSettings.FromDefaultSources();
+
+// Accessing settings for an unknown integration will log a warning
+var settings = tracerSettings .Integrations["MyRandomIntegration"];
+
+// changes are not persisted
+settings.Enabled = false; 
+
+// isEnabled is null, not false
+bool? isEnabled = tracerSettings.Integrations["MyRandomIntegration"].Enabled;
+```
+
+### Default value of `DD_TRACE_ROUTE_TEMPLATE_RESOURCE_NAMES_ENABLED` is now `true`
+
+Version 1.26.0 added support for the `DD_TRACE_ROUTE_TEMPLATE_RESOURCE_NAMES_ENABLED` feature flag, which enabled improved span names for ASP.NET and ASP.NET Core automatic instrumentation spans, an additional span for ASP.NET Core requests, and additional tags. 
+
+In version 2.0, `DD_TRACE_ROUTE_TEMPLATE_RESOURCE_NAMES_ENABLED` is **enabled** by default. Due to the change in span names, you may need to update your monitors and dashboards to use the new resource names. 
+
+If you do not wish to take advantage of the improved route names, you can disabled the feature by setting the `DD_TRACE_ROUTE_TEMPLATE_RESOURCE_NAMES_ENABLED` environment variable to `0`.
+
+### CallSite instrumentation has been removed
+
+From version 2.0, callsite automatic instrumentation has been removed and replaced with calltarget instrumentation. This has been the default mode [since version 1.28.0](https://github.com/DataDog/dd-trace-dotnet/releases/tag/v1.28.0). Calltarget provides performance improvements over callsite, but we no longer support instrumenting some custom implementations of `DbCommand`. If you find spans are missing from your traces after upgrading, please raise an issue on GitHub, or contact [support](https://docs.datadoghq.com/help).
+
+### Integrations.json has been removed
+
+The _integrations.json_ file is no required for instrumentation. You can remove references to this file, for example by deleting the `DD_INTEGRATIONS` environment variable.
```

Comment:
```suggestion
The `integrations.json` file is no required for instrumentation. You can remove references to this file, for example by deleting the `DD_INTEGRATIONS` environment variable.
```

Reply:
This is one of my pet peeves: `integrations.json` isn't code, it shouldn't be in a "code" block 😉 But if this is standard and we do the same in our docs for example, then consistency is obviously more important 😛

Reply:
`The `integrations.json` file is no required for instrumentation` 
The `integrations.json` file is no **longer** required for instrumentation?  😃

Reply:
Adding Anna's suggestion as a GitHub suggestion (including the code block):
```suggestion
The `integrations.json` file is no longer required for instrumentation. You can remove references to this file, for example by deleting the `DD_INTEGRATIONS` environment variable.
```